deploy/handler/cluster_role.yaml

```diff
@@ -0,0 +1,19 @@
+{{if .ClusterReaderExists}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nmstate-cluster-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
+rules:
+- apiGroups:
+  - nmstate.io
+  resources:
+  - nodenetworkstates
+  - nodenetworkconfigurationpolicies
+  - nodenetworkconfigurationenactments
+  verbs:
+  - get
+  - list
+  - watch
+{{end}}
```

Comment:
What do you think about removing this from the "role.yaml" and create another role binding to bind this role to the nmstate-handler user ? this way we don't need the conditional since we must deploy it.

Reply:
We need to allow the cluster-reader to read those, not other user, so it won't help
and imo self contained like this
unless I miss anything ?